Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.8.8-buster AS base
+RUN apt-get update && apt-get install -y curl
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+EXPOSE 5000
+ENV PATH="${PATH}:/root/.poetry/bin"
+RUN mkdir -p /devops_mod5
+WORKDIR /devops_mod5
+COPY . /devops_mod5
+RUN poetry install 
+ENV VIRTUAL_ENV=.venv
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
```

Comment:
You should be able to remove these lines if you run gunicorn with `poetry run gunicorn`.

But better yet, you don't need a virtual environment inside your Docker container because it's already isolated. So you could instead use this for your production stage 
```
RUN poetry config virtualenvs.create false --local && poetry install --no-dev --no-root
```
Note the `--no-dev` flag is to skip packages like `pytest` assuming the pyproject.toml file is set up correctly.

Reply:
Thanks Hugh
Have removed the virtual env ENV config and added the suggested RUN line